samples/embedded-auth-with-sdk/src/main/java/com/okta/spring/example/controllers/LoginController.java

```diff
@@ -286,6 +356,44 @@ public ModelAndView verify(final @RequestParam("code") String code,
         return responseHandler.handleKnownTransitions(authenticationResponse, session);
     }
 
+    /**
+     * Handle webauthn authenticator verification functionality.
+     *
+     * @param requestBodyJson
+     * @param session the session
+     * @return the view associated with authentication response.
+     */
+    @PostMapping("/verify-webauthn")
+    public ModelAndView verifyWebAuthn(final @RequestBody String requestBodyJson,
+                                       final HttpSession session) {
+        logger.info(":: Verify Webauthn ::");
+
+        ProceedContext proceedContext = Util.getProceedContextFromSession(session);
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        JsonNode jsonNode = null;
+        try {
+            jsonNode = objectMapper.readTree(requestBodyJson);
+        } catch (JsonProcessingException e) {
+            logger.error("Error occurred:", e);
+        }
+
+        String clientData = jsonNode.get("clientData").asText();
```

Comment:
Is there a way to make spring do this parsing instead of having to do it inline?

Reply:
yep, will refine this.

Reply:
updated PR.